puphpet/config.yaml.template

```diff
@@ -60,10 +71,20 @@ vagrantfile-local:
         host: detect
 server:
     install: '1'
+    apt-ppa-repos:
+        - ppa:git-core/ppa   # Provides newer git-core. -bp
+        - ppa:chris-lea/node.js  # Provides newer node and npm. -bp
     packages:
-        - vim
-        - memcached
         - curl
+        - emacs
```

Comment:
Anytime you have to destroy and rebuild it's inconvenient and time consuming. I'm not in favor of punishing devs because they or someone else didn't think to immediately remove things from the skeleton. Every time I typically rebuild is time consuming because it's been quite some time since the last time I did it.

I'd hate to write anything of puppet.com into the skeleton.

Suggesting packages is fine. Installing them because a dev ran `vagrant up` too quickly is not.

Are all of these software packages in every production instance? If so that's fine, but it doesn't seem "right" to me. I don't think I'll even be convinced that more software is the answer. I also don't believe that Vagrant should be used as a way to create production boxes. I don't see it as a sysadmin tool, rather solely a development tool, at least in it's current state.